src/appComponent.js

```diff
@@ -27,31 +32,47 @@ class AppComp extends Component {
     this.setState({ user });
   }
 
+  onLogout = () => {
+    this.setState({ user: null });
+  }
+
   markDone = (data) => {
```

Comment:
It shouldn't be here. This method should be created in component which uses it (the same for `delTask`)